src/odemis/gui/model/__init__.py

```diff
@@ -876,6 +876,26 @@ def _on_project_path_change(self, _):
                     config.pj_last_path, config.fn_ptn,
                     config.last_extension,
                     config.fn_count)
+        
+class CryoCorrelationGUIData(CryoGUIData):
+    """ Represent an interface used to correlate multiple streams together.
+
+    Used for METEOR systems.
+
+    """
+
+    def __init__(self, main):
+        super().__init__(main)
+
+        # Current tool selected (from the toolbar)
+        tools = {TOOL_NONE}
+        # Update the tool selection with the new tool list
+        self.tool.choices = tools
+
+        # for export tool
+        self.acq_fileinfo = VigilantAttribute(None) # a FileInfo
```

Comment:
I think this is to display "the" file information (name/date) in the analysis tab. Here, there are typically at least 2 files opened, and the filename doesn't really mater. => Drop the display (if there is) and drop this VA?

Reply:
This is required to use the existing export tool, it checks for this variable for a potential filename, but if its None, just uses the datetime. So needs to exist, but can be None

https://github.com/patrickcleeve2/odemis/blob/372028261d6b29a5dbab538856a7e1ffdf556dfa/src/odemis/gui/cont/export.py#L141-L142